src/main/resources/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfig/help-deleteBlobs.html

```diff
@@ -1 +1,3 @@
-<div>this parameter is set by the property <code>-Dio.jenkins.plugins.artifact_manager_s3.S3BlobStoreConfig.deleteBlobs</code> enable deleting blobs when a build is deleted</div>
\ No newline at end of file
+<div>This parameter is set by the property <code>-Dio.jenkins.plugins.artifact_manager_s3.S3BlobStoreConfig.deleteBlobs</code>
+    ,if not checked the blobs will not be deleted from S3 when the corresponding build is deleted.
```

Comment:
I do not think we want to refer to “blobs” in the UI, especially in contrast to “stashes”: as far as S3 is concerned, they are all blobs, whereas Jenkins is distinguishing “artifacts” from “stashes”.

In fact we may want to rename the API methods to use the `deleteArtifacts` naming pattern.